wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/DeviceDetailsViewActionsHandler.swift

```diff
@@ -130,6 +138,39 @@ final class DeviceDetailsViewActionsHandler: DeviceDetailsViewActions, Observabl
         }
         return fingerPrint.splitStringIntoLines(charactersPerLine: 16).uppercased()
     }
+
+    private func startE2EIdentityEnrollment() async throws {
+        typealias E2ei = L10n.Localizable.Registration.Signin.E2ei
+        let e2eiCertificateUseCase = userSession.enrollE2eICertificate
+        guard let rootViewController = await AppDelegate.shared.window?.rootViewController else {
+            return
+        }
+        let oauthUseCase = OAuthUseCase(rootViewController: rootViewController)
+        try await e2eiCertificateUseCase?.invoke(
+            authenticate: oauthUseCase.invoke
+        )
+        userSession.reportEndToEndIdentityEnrollmentSuccess()
```

Comment:
I don't think we should call `reportEndToEndIdentityEnrollmentSuccess()`.  It was relevant for the login flow.

Reply:
done